packages/design-system/src/components/Pagination/Pagination.tsx

```diff
@@ -0,0 +1,294 @@
+      /**
+       * If `pageRange` begins with a page of 2 or greater,
+       * begin Pagination with Page 1
+       */
+      pages.push(
+        <Page
+          customUrl={customUrl}
+          key="page-1"
+          index={1}
+          isActive={currentPage === 1}
+          onPageChange={pageChange(1)}
+        />
+      );
+
+      /**
+       * If `pageRange` doesn't equal 2, second Pagination element is Ellipses,
+       * otherwise page count continues.
+       */
+      if (pageRange[0] !== 2) {
+        pages.push(<Ellipses key="ellipses-1" />);
+      }
+    }
+
+    /**
+     * Renders all Page components in range (3 pages) to Pagination component.
+     */
+
+    pageRange.map((page) => {
+      pages.push(
+        <Page
+          customUrl={customUrl}
+          key={`page-${page}`}
+          index={page}
+          isActive={currentPage === page}
+          onPageChange={pageChange(page)}
+        />
+      );
+    });
+
+    /**
+     * Defines if/when the Ellipses component appears
+     * at the end of the Pagination component -
+     * as long as there are fewer than 7 pages.
+     */
+    if (currentPage <= totalPages - 3 && totalPages > maxVisiblePages) {
+      if (currentPage < totalPages - 3) {
+        pages.push(<Ellipses key="ellipses-2" />);
+      }
+
+      pages.push(
+        <Page
+          customUrl={customUrl}
+          key={`page-${totalPages}`}
+          index={totalPages}
+          isActive={currentPage === totalPages}
+          onPageChange={pageChange(totalPages)}
+        />
+      );
+    }
+  }
+
+  return (
+    <nav className={classes} role="navigation" aria-label="Pagination Navigation">
+      {currentPage !== 1 && (
+        <Button
+          variation="tertiary"
+          href={customUrl ? `${customUrl}/${currentPage - 1}` : `#${currentPage - 1}`}
+          onClick={pageChange(currentPage - 1)}
+        >
+          <span className="ds-c-pagination__nav--previous-img-container">
+            <svg
+              aria-hidden="true"
+              focusable="false"
+              data-prefix="fas"
+              data-icon="chevron-left"
+              className="ds-c-pagination__nav--image"
+              role="img"
+              xmlns="http://www.w3.org/2000/svg"
+              viewBox="0 0 320 512"
+            >
+              <path
+                fill="currentColor"
+                d="M34.52 239.03L228.87 44.69c9.37-9.37 24.57-9.37 33.94 0l22.67 22.67c9.36 9.36 9.37 24.52.04 33.9L131.49 256l154.02 154.75c9.34 9.38 9.32 24.54-.04 33.9l-22.67 22.67c-9.37 9.37-24.57 9.37-33.94 0L34.52 272.97c-9.37-9.37-9.37-24.57 0-33.94z"
+              />
+            </svg>
+          </span>
+          {leftLabel}
+        </Button>
+      )}
+
+      {isMobile || compact ? (
+        <span className="ds-c-pagination__page-count">
+          Page <strong>{currentPage}</strong> of <strong>{totalPages}</strong>
+          <span
+            className="sr-only"
+            aria-live="assertive"
```

Comment:
The Axe checker throws an error here.  See: https://dequeuniversity.com/rules/axe/4.3/aria-allowed-attr I think because of the combination of `aria-live` and `aria-label`. It looks like you're trying to make the wording more specific, yeah? I think Page # of # is enough. Why the `aria-live`? Is it because this will likely update without a page refresh?

Reply:
`aria-live` was me goofin'. I've refactored it to just include ARIA label on the parent span element.